README.md

```diff
@@ -331,7 +331,10 @@ as follows:
 
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
-    home-manager.url = "github:nix-community/home-manager";
+    home-manager = {
+      url = github:nix-community/home-manager;
+      inputs.nixpkgs.follows = "nixpkgs";
```

Comment:
Strictly speaking, this is only necessary under the following circumstances:
- when using the `lib.hm` output to pin the `nixpkgs` that it uses (`lib.hm` should have been a function that accepts a `pkgs` to allow using Home Manager's lib with different nixpkgs input)
- when using Home Manager standalone (not as a NixOS or Darwin module)

When using Home Manager as a module, the `home-manager.useGlobalPkgs` is required to be true, which makes Home Manager use whatever `pkgs` is used in the context of evaluating the (NixOS/Darwin) configuration, thus making the `follows` code unnecessary.